README.md

```diff
@@ -105,8 +105,15 @@ For more instructions about adding and configuring cactbot overlays, see the [Ad
 
 Follow all the steps above for installing cactbot first.
 
-1. Follow the instructions in the **dummy.txt** file in [plugin/ThirdParty/OverlayPlugin](plugin/CactbotOverlay/ThirdParty/OverlayPlugin).
-1. Follow the instructions in the **dummy.txt** file in [plugin/ThirdParty/ACT](plugin/CactbotOverlay/ThirdParty/ACT).
+##### Precondition:
+1. `curl` MUST be installed (this is used to download dependencies)
```

Comment:
What version of curl is required?

I get this error: ```curl: (35) error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version```

Reply:
currently i use following `curl 7.59.0 (x86_64-w64-mingw32) libcurl/7.59.0 OpenSSL/1.0.2h (WinSSL) zlib/1.2.8 WinIDN libssh2/1.7.0_DEV
Release-Date: 2018-03-14`
i just put that info into there because i saw @ngld uses curl in his files, maybe he is can better answer, what version really is required or if there is a workaround

Reply:
Any recent version should do. That error most likely indicates that your curl build doesn't support TLS 1.2. That seems odd... Did you get your build from [the official website](https://curl.haxx.se/download.html#Win64)?

If you have Windows 10, you can use the preinstalled curl so this shouldn't be an issue in the future.

Reply:
I apparently was running on an ancient version of git bash.  Updating fixed this unsurprisingly.